### DISCUSSION TITLE
Custom Elastic Syslog Parsing working in Elasticsearch, but not showing as parsed in Hunt

### DISCUSSION THREAD

**Discussion by luke879** [Security-Onion-Solutions/securityonion#9793] [Unsupported Versions]
![ElasticVsHuntLogs](https://user-images.githubusercontent.com/13888807/219175961-63a5dc55-4ab8-4b8b-ab7c-34dc94d3e7cb.png)
Looking for assistance to synchronize hunt fields and parsing to match ElasticSearch fields. My team implemented a custom Apereo CAS syslog parser with GROK etc that works correctly in ElasticSearch, but unt shows the unparsed content, and is not showing some of the populated fields.  Looking at an identical record, elasticsearch fields user.name and what do not show in hunt. source.application in Elasticsearch shows correctly after the grok/regex, but shows incorrectly (as preparsed) in hunt. 

Any assistance would be greatly appreciated.

Thanks!

**Comment by cm-ops** [comment]
Kibana and SOC are both querying the Elasticsearch indices. Are you able to share the same document side by side? Those are two different logs looking at the timestamp.
